frontend/src/scenes/insights/ActionFilter/entityFilterLogic.ts

```diff
@@ -20,7 +20,7 @@ export function toLocalFilters(filters: Partial<FilterType>): LocalFilter[] {
         .sort((a, b) => a.order - b.order)
         .map((filter, order) => ({ ...(filter as ActionFilter), order }))
     return localFilters.map((filter) =>
-        filter.properties
+        filter.properties && Array.isArray(filter.properties)
```

Comment:
I'm confused now 😅 , if it's an array, why would we call `convertPropertyGroupToProperties` ? Is it to filter out invalid properties only?

Reply:
Yeah so we don't call `filter` from that util on an empty object

Reply:
The method only takes in these three types `properties: PropertyGroupFilter | AnyPropertyFilter[] | undefined` and since this is an edge case of forced typing we should just filter it out before it gets to the method vs including another type